spec/gitlab/cli_spec.rb

```diff
@@ -59,6 +60,21 @@
       end
     end
 
+
+    context "when command with json output" do
+      before do
+        stub_get("/user", "user")
+        args = ['user', '--json']
+        @output = capture_output { Gitlab::CLI.start(args) }
+      end
+
+      it "should render output as json" do
+        expect(JSON.parse(@output)['result']).to eq(JSON.parse(File.read(File.dirname(__FILE__) + '/../fixtures/user.json')))
```

Comment:
I'm not an expert at writing tests, so don't assume I know what I'm talking about here, but I think maybe you should check here that some of the individual attributes (email, id, etc.) are correct instead.